src/video.jakt

```diff
@@ -1,20 +1,36 @@
-import sdl { SDL }
+import extern c "SDL.h" {}
+import extern "algorithm" {}
+import sdl { SDL,  null }
 import ppu { RGB } 
 
 function draw_frame(sdl: SDL, buffer: [RGB]) throws {
-    let WINDOW_WIDTH = 600;
-    mut user_x = 100u64;
-    mut user_y = 100u64;
 
-    for i in 0..(256u64 * 240u64) {
-        let rgb = buffer[i]
-        sdl.set_render_draw_color(red: rgb.red, green: rgb.green, blue: rgb.blue, alpha: 255)
+    mut format = 0u32
+    mut access: c_int = 0
+    unsafe {
```

Comment:
This makes me wonder if we want to extend sdl.jakt to be able to create and draw its own frames.

That way we can keep all the SDL.h stuff in a single file. We may also want to move our RGB palette to being u32 already so we don't have to do the conversion step.